projects/control-service/projects/job-builder-secure/Dockerfile.python.vdk

```diff
@@ -1,45 +1,58 @@
 # https://docs.docker.com/develop/develop-images/dockerfile_best-practices
 
 ARG base_image=versatiledatakit/data-job-base-python-3.10-secure:latest
-
 FROM $base_image
 
 ARG UID=1000
 ARG GID=1000
+ARG job_name
+ARG job_githash
+ARG requirements_file=requirements.txt
+
+ENV JOB_NAME $job_name
+ENV VDK_JOB_GITHASH $job_githash
+ENV HOME=/job
 
 # Set the working directory
 WORKDIR /job
 
-# Validate base image is python based
-RUN python -V
-# Create necessary users and set home directory to /job
-RUN groupadd -r -g $GID group && useradd -u $UID -g $GID -r user && chown -R $UID:$GID /job
-ENV HOME=/job
-
 # Copy the actual job that has to be executed
-ARG job_name
 COPY --chown=$UID:$GID $job_name $job_name/
 
-# Remove execute permissions for files within the Data job directory, but not for the directories themselves
-RUN chmod -R -x+X $job_name/*
-
-# TODO: this would trigger for any change in job even if requirements.txt does not change
-# but there's no COPY_IF_EXISTS command in docker to try copy it.
-ARG requirements_file=requirements.txt
-RUN if [ -f "$job_name/$requirements_file" ]; then pip3 install --disable-pip-version-check -q -r "$job_name/$requirements_file" || ( echo ">requirements_failed<" && exit 1 ) ; fi
-
-ARG job_githash
-ENV JOB_NAME $job_name
-ENV VDK_JOB_GITHASH $job_githash
-
-# Delete system executables
-RUN rm /usr/bin/chmod
-RUN rm /usr/bin/chown
-RUN rm /usr/sbin/groupadd
-RUN rm /usr/sbin/groupdel
-RUN rm /usr/sbin/useradd
-RUN rm /usr/sbin/userdel
-RUN rm /usr/bin/uname
-RUN python -m pip uninstall pip -y
+# Install native dependencies
+RUN : \
```

Comment:
nit, wouldn't it be better to move it to a shell script "build_job.sh" which you can add and RUN 

Might be more readable

Reply:
I will keep it as it is for the time being.